p2p_wallet/Scenes/Main/BankTransfer/BankTransferInfoViewModel.swift

```diff
@@ -0,0 +1,150 @@
+import BankTransfer
+import CountriesAPI
+import Combine
+import SwiftUI
+import Resolver
+import KeyAppUI
+
+final class BankTransferInfoViewModel: BaseViewModel, ObservableObject {
+
+    // MARK: -
+
+    var showCountries: AnyPublisher<([Country], Country?), Never> {
+        showCountriesSubject.eraseToAnyPublisher()
+    }
+
+    // MARK: - Dependencies
+
+    @Injected private var bankTransferService: BankTransferService
+    @Injected private var countriesService: CountriesAPI
```

Comment:
so you left it here? and we don't need it UserData? can we remove a field from there?